packages/playground/src/components/swagger-ui.tsx

```diff
@@ -0,0 +1,22 @@
+import { useEffect, useRef } from "react";
+import { FunctionComponent } from "react";
+import CreateSwaggerUI from "swagger-ui";
+export interface SwaggerUIProps{
+  spec: string; 
+}
+export const SwaggerUI: FunctionComponent<SwaggerUIProps> =(props)=> {
+  const uiRef = useRef(null);
+  const uiInstance = useRef<any | null >(null);
```

Comment:
`any | null` is just `any` I think, so if this can't be typed more accurately just use `any` I think.